pkg/metrics/util.go

```diff
@@ -22,3 +23,33 @@ func EventPayload(ev types.Event) ([]byte, error) {
 	payload := map[string]interface{}{"event": ev, "versions": vmap}
 	return json.Marshal(payload)
 }
+
+var secretKeywords = []string{"password", "secret", "token", "key"}
+
+// redactFlags redacts presumed secret values from a string slice based on preset keywords.
+// For example, given ["--license", "./path", "--password", "123"], it will return ["--license", "./path", "--password", "*****"]
+func redactFlags(flags []string) []string {
```

Comment:
i dont think you handle the case where the flag is used with an equals sign

```
--admin-console-password=*****
```

Reply:
Good point, let me rework this.

Reply:
Fixed via - https://github.com/replicatedhq/embedded-cluster/pull/1528/commits/9a65e095c4760737bd0e47ca847cc2440b0a7a4d